nextflow.config

```diff
@@ -0,0 +1,595 @@
+// include parameters in params.config
+includeConfig "$projectDir/conf/params.config"
+includeConfig "$projectDir/conf/resources.config"
+
+// executor
+process.executor       = 'slurm'
```

Comment:
Setting the executor should be a user specific configuration, rather than a pipeline config. There's no way of knowing what the infrastructure is like for the end user. They might want to run this on the cloud or using an sge cluster instead.